src/pages/html/elements/tr-tag/index.md

```diff
@@ -1,15 +1,34 @@
 ---
 title: Tr Tag
 ---
-## Tr Tag
+<p>The <code class="language-text">&lt;tr&gt;</code> tag defines a standard row in an HTML table on a web page.</p>
+<p>The <code class="language-text">&lt;tr&gt;</code> tag houses the <code class="language-text">&lt;th&gt;</code> and <code class="language-text">&lt;td&gt;</code> tags, also known as Table Heaeder and Table Data respectively.</p>
```

Comment:
Typo on line 5: Heaeder

Reply:
Thanks for noticing. Fixed it.

Reply:
Np, you should also change the title as mentioned in the guide.